### PR TITLE
githubteacher's tampa food list

### PR DESCRIPTION
**Pull request by githubteacher**
Added restaurants that I suggest going to. Used a bunch of emoji. Probably should add gifs at some point.

@brianamarie can you check this out, is there any other information you need?

### CODE REVIEW

**Comment by brianamarie** [comment]
Looks good!...any good taco places?

File: githubteacher-tampa.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+## Tampa, Fl
+
+### Places to eat :hamburger:
+- Tarpon Tavern for pub food and :beer:
```
Comment (mpgrill) [inline githubteacher-tampa.md:4]: Sounds great!

**Comment by RMBloom** [comment]
@githubteacher Nothing from Clearwater?

File: githubteacher-tampa.md (hunk 1 above)
Comment (githubteacher) [reply githubteacher-tampa.md:4]: @mpgrill 👍

Comment (vicmaster) [reply githubteacher-tampa.md:4]: If you want some 🌮  come to https://github.com/githubteacher/dev-oct-13/pull/45

**Pull request merged**